.github/workflows/unit-tests-integrations.yaml

```
@@ -54,6 +51,9 @@ jobs:
       - name: Prepare workspace
         uses: ./.github/actions/prepare-workspace
 
+      - name: Build teleport binaries
+        run: make && echo "$PWD/build" >> "$GITHUB_PATH"
```

If you don't need `tsh` or `tbot`, you could save some time with:
```suggestion
        run: make build/teleport build/tsh && echo "$PWD/build" >> "$GITHUB_PATH"
```